.github/workflows/test.yml

```diff
@@ -1,18 +1,19 @@
 name: test
 on: [pull_request, push]
 
+env:
+  MIN_LINE_RATE: 0.2
+
 jobs:
   run_tests:
     runs-on: self-hosted
+    container:
+      image: python:3.8
```

Comment:
change these to `python:3.10`? see https://github.com/MuttData/muttlib/pull/6

Reply:
just in case let's do this in that PR